tools/cmd.go

```diff
@@ -67,3 +70,33 @@ func mustGetDuration(cmd *cobra.Command, flagName string) time.Duration {
 	}
 	return val
 }
+
+func ResolveManifestFile(input string) (manifestName string, err error) {
+	if input == "" {
+		_, err := os.Stat("substreams.yaml")
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				return "", fmt.Errorf("no manifest entered in dir w/o a manifest")
+			}
+			return "", fmt.Errorf("finding manifest: %w", err)
+		}
+
+		return "substreams.yaml", nil
+	}
+
+	inputInfo, err := os.Stat(input)
+	if err != nil {
+		return "", fmt.Errorf("reading input: %w", err)
+	}
+
+	// If input is dir
+	if inputInfo.IsDir() {
+		potentialManifest := filepath.Join(inputInfo.Name(), "substreams.yaml")
+		_, err := os.Stat(potentialManifest)
+		if err != nil {
+			return "", fmt.Errorf("finding manifest in dir: %w", err)
+		}
+		return "substreams.yaml", nil
```

Comment:
That's seems quite wrong ... If the input was a directory, so maybe `"/tmp/some/path"` we return `substreams.yaml` which points to current working directory, have you tested all cases properly?

Reply:
yea you're right.